storage/src/main/java/tech/pegasys/teku/storage/server/pruner/BlobSidecarPruner.java

```diff
@@ -81,6 +85,9 @@ public BlobSidecarPruner(
     this.pruningActiveLabelledGauge = pruningActiveLabelledGauge;
     this.storeNonCanonicalBlobSidecars = storeNonCanonicalBlobSidecars;
 
+    // To be updated with other implementations. e.g. filesystem or s3
```

Comment:
I'd use `// TODO: ` prefix so it will be captured nicely by intellij, or simply remove since we have a (set of) issues on the topic.

Reply:
I'll fix that up in the next PR. Thanks for feedback.